docker/README.md

```diff
@@ -11,3 +11,34 @@ The following table provides a list of Dockerfiles that you can use to build the
 | `merlin-hugectr`     | `dockerfile.ctr`   | <https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-hugectr>    | NVTabular and HugeCTR                                          |
 | `merlin-tensorflow`  | `dockerfile.tf`    | <https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-tensorflow> | NVTabular, TensorFlow, and HugeCTR Tensorflow Embedding plugin |
 | `merlin-pytorch`     | `dockerfile.torch` | <https://catalog.ngc.nvidia.com/orgs/nvidia/teams/merlin/containers/merlin-pytorch>    | NVTabular and PyTorch                                          |
+
+
+# Building the Merlin containers locally
```

Comment:
Please use an H2, `##`.  Up to you on whether to increment the others: it doesn't seem critical to preserve the hierarchy and flatter is usually easier to scan.

Please also use title case for all the headings.